include/download_client.h

```diff
@@ -67,6 +67,18 @@ struct download_client_evt {
 	};
 };
 
+/**
+ * @brief Download client TLS and APN configuration options.
```

Comment:
`Download client configuration options.` perhaps? Just in case it's extended in the future. No need to repeat ourselves, you can just check the structure content.

Reply:
Ok, rewritten.